### ISSUE TITLE
dUnzip2: File 'manifest.xml' is not compressed in the zip

### ISSUE THREAD

**Issue by Darshan0003** [sardoj/VTModuleDesigner#114]
Please Suggest a solution for the following error while installing the patch "VTModuleDesigner-vt6.1".
Have replaced all the files and folders and then imported the module.

dUnzip2: File 'manifest.xml' is not compressed in the zip.
Warning: simplexml_load_file(): I/O warning : failed to load external entity "test/vtlib/manifest-1433310578.xml" in F:\vtiger\vtigerCRM620\apache\htdocs\vtigerCRM\vtlib\Vtiger\PackageImport.php on line 54 Warning: unlink(test/vtlib/manifest-1433310578.xml): No such file or directory in F:\vtiger\vtigerCRM620\apache\htdocs\vtigerCRM\vtlib\Vtiger\PackageImport.php on line 55
FailedInvalid File provided for module import! Try Again.

**Comment by matteoraggi** [comment]
I have a similar problem with vtiger 6.3:
dUnzip2: File 'manifest.xml' is not compressed in the zip.

## FailedInvalid File provided for module import! Try Again.

Did you patched your problem in the while?

**Comment by fauners** [comment]
Unzip download archive, and compress only the SRC folder and upload.

**Comment by matteoraggi** [comment]
Thanks! Done and then I receive the same error (my hosting is fastcomet): 
dUnzip2: File 'manifest.xml' is not compressed in the zip.
FailedInvalid File provided for module import! Try Again

**Comment by fauners** [comment]
Here is the one I used and it worked fine. This is just the contents of the src folder in a zip
http://www.filedropper.com/vtmoduledesigner

**Comment by matteoraggi** [comment]
YEah! with this it works! so maybe you have to update the github file or publish the version for vtiger 6.3?
ModuleDesigner
Vtiger Version : 6.1.0
 I accept the license agreement.

**Comment by beingprabhu** [comment]
@fauners, I tried to follow the steps which you have written. But nothing worked. The link which you have posted in your comment is not showing anything except filedropper home page.. So can you please re post vtmoduledesigner zip file in filedropper so that i can download it.. thanks in advance

**Comment by fauners** [comment]
Hi
I simply unzipped the downloaded file from git hub and went into the
folder,  think it's called module.
You will see the manifest file in it.
Select all of those files/folders and zip them and upload. Not the folder
that contains them but the files themselves.
This worked for me.
If it doesn't work I'll be in work tomorrow and can upload the file again

**Comment by beingprabhu** [comment]
![untitled](https://cloud.githubusercontent.com/assets/15322175/10758313/bad3fc8a-7cd6-11e5-81c6-986377731bd0.jpg)

@fauners, thanks for the reply bro.. Just for testing purpose I created a demo module.. Now I want to add/edit the fields I've entered. I went to module manager, on clicking settings I got Edit Fields. It redirected me to " Fields and Layout Editor " page and there I'm not finding any blocks which I created.
My question is, is that a bug or a problem in installation or is there any other way. Since I'm very new to CRMs, I'm not able understand it.. I'm not getting any useful guide or tutorial for using module designer. So I'm seeking for your help... If you have any study material or any useful thing(links) that can help me in.. Please share it with me and by that other people can find useful.. Thanks in advance
